ecosystem/indexer-grpc/transaction-filter/src/utils.rs

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+// 64 "0"s
+const ZEROS: &str = "0000000000000000000000000000000000000000000000000000000000000000";
+
+/// Converts a "0x" prefixed address to display format (short for special addresses, long for all other addresses):
+/// https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-40.md#display-format
+#[inline]
+pub fn standardize_address(address: &str) -> String {
+    // Remove "0x" prefix
+    let trimmed = &address[2..];
```

Comment:
Only remove the prefix if present, relaxed parsing allows for account addresses without the 0x prefix.